.github/workflows/builds.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-latest, macos-10.15]
+        os: [macos-latest, macos-11]
```

Comment:
The `macos-latest` virtual environment is currently [defined](https://github.com/actions/virtual-environments#available-environments) as macOS 11.  Maybe we should be more direct with `[macos-11, macos-12]`?

Reply:
Good suggestion.